eclipsePlugin/build.gradle

```diff
@@ -34,6 +36,12 @@ def eclipsePluginId = 'com.github.spotbugs.plugin.eclipse'
 def localProps = new Properties()
 localProps.load(new FileInputStream("$projectDir/local.properties"))
 
+ext {
+  eclipseRootDir = localProps.getProperty('eclipseRoot.dir')
+  eclipseExecutable = eclipseRootDir + (Os.isFamily(Os.FAMILY_MAC) ? "/Contents/MacOS/eclipse" : "/eclipse")
+  eclipsePluginsDir = eclipseRootDir + (Os.isFamily(Os.FAMILY_MAC) ? "/Contents/Eclipse/plugins" : "/plugins")
+}
```

Comment:
Please note that SpotBugs *requires* Java 8. So if someone would try to install it on Eclipse before Luna (4.4) it will most likely fail. I doubt there are many using Kepler with additional JDT patch for Java 8, so first really useful Eclipse verson for Java 8 is 4.4 (Luna).

Also there is still FindBugs plugin which supports older Eclipse installations.

So for me we can target Eclipse >= 4.5. Everyone who want to use SpotBugs plugin with older Eclipse versions should clone the code, create a patch and use it.